realsense_camera/launch/includes/nodelet.launch.xml

```diff
@@ -22,18 +22,9 @@
     <param name="ir_frame_id"             type="str"  value="$(arg camera)_ir_frame" />
     <param name="ir2_frame_id"            type="str"  value="$(arg camera)_ir2_frame" />
 
-    <remap from="camera/depth/image_raw"    to="$(arg camera)/$(arg depth)/image_raw" />
-    <remap from="camera/color/image_raw"    to="$(arg camera)/$(arg rgb)/image_raw" />
-    <remap from="camera/ir/image_raw"       to="$(arg camera)/$(arg ir)/image_raw" />
-    <remap from="camera/ir2/image_raw"      to="$(arg camera)/$(arg ir2)/image_raw" />
-    <remap from="camera/depth/points"       to="$(arg camera)/$(arg depth)/points" />
-    <remap from="camera/depth/camera_info"  to="$(arg camera)/$(arg depth)/camera_info" />
-    <remap from="camera/color/camera_info"  to="$(arg camera)/$(arg rgb)/camera_info" />
-    <remap from="camera/ir/camera_info"     to="$(arg camera)/$(arg ir)/camera_info" />
-    <remap from="camera/ir2/camera_info"    to="$(arg camera)/$(arg ir2)/camera_info" />
-    <remap from="camera/get_settings"       to="$(arg camera)/get_settings" />
-    <remap from="camera/set_power"          to="$(arg camera)/set_power" />
-    <remap from="camera/force_power"        to="$(arg camera)/force_power" />
-    <remap from="camera/is_powered"         to="$(arg camera)/is_powered" />
+    <remap from="depth"    to="$(arg depth)" />
```

Comment:
Yup, it does.  Give it a try -- try adding  "<arg name="depth" value="foo" />" to r200_nodelet_default.launch (or any other launch file that includes nodelet.launch.xml), and you'll see that depth gets appropriately remapped to foo, within the camera group namespace.

Basically, no, we shouldn't be handling the topic remapping ourselves in code -- we should be publishing and subscribing to our published API, and let the ROS name remapping tools handle the remapping for us should a user choose to use it.  That's a large part of what this whole change is about -- by not hardcoding the topic names, this kind of remapping actually works.